src/styles/main_page/_Common.scss

```diff
@@ -12,13 +13,20 @@
 
 .heading-container {
   .heading {
-    font-size: 3.5rem;
-    margin-bottom: 0.8rem;
+    font-size: 4.5rem;
+    margin-bottom: -0.8rem;
+    font-weight: 100;
+    &.white {
+      color: $white;
+    }
   }
 
   .heading-line {
-    width: 10rem;
+    width: 8rem;
     height: 2px;
     background-color: $black;
+    &.white {
+      background-color: $white;
+    }
```

Comment:
If not necessary, try not to change Common.scss. This might break other people's style.